src/MPDevice.h

```diff
@@ -393,7 +393,7 @@ private slots:
     quint32 credentialsDbChangeNumberClone;
     quint32 dataDbChangeNumberClone;
     QByteArray ctrValueClone;
-    QVector<QByteArray> startNodeClone = {0};
```

Comment:
oh i actually wondered about that. why the two elements though?

Reply:
To make sure not causing an issue with webauthn creds. In theory it should not, but after the mini crashes I rather made sure of it.